source/stylesheets/elements/_typography.scss

```diff
@@ -0,0 +1,46 @@
+html {
+  font-family: $tbds-font-sans-serif;
+  font-size: 100%;
+  line-height: 1.6;
+
+  @media (min-width: 38em) {
```

Comment:
Would it be beneficial to have these breakpoints recorded as variables? Do we use them/will we use them anywhere else? IMO just naming them might give a little more context as to what's happening at each one

Reply:
Yes! I think that is a very good idea.

These particular styles are not new in this PR, but were brought over from `base/_typography.scss`, so I didn’t concern myself with refactoring them right now.

Very relevant to your point, we should soon have breakpoint variables and mixins available from the design system: https://github.com/thoughtbot/design-system/pull/67